pages/exercises/[lessonSlug].tsx

```diff
@@ -25,12 +26,45 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
 
   return (
     <Layout title={currentLesson.title}>
+      <ExercisesNavCard lessonDocUrl={currentLesson.docUrl} lessonSlug={slug} />
       <h1>{currentLesson.title}</h1>
       {alerts && <AlertsDisplay alerts={alerts} />}
     </Layout>
   )
 }
 
+type ExercisesNavCardProps = {
+  lessonDocUrl: string | null | undefined
+  lessonSlug: string
+}
+
+const ExercisesNavCard = ({
+  lessonDocUrl,
+  lessonSlug
+}: ExercisesNavCardProps) => {
+  return (
+    <div className="card shadow-sm d-inline-flex flex-row p-2 mb-4">
+      {lessonDocUrl && (
+        <NavLink
+          path={lessonDocUrl}
+          className="py-2 px-3 mx-1 fw-bold text-primary"
+        >
+          LESSONS
+        </NavLink>
+      )}
+      <NavLink
+        path={`/curriculum/${lessonSlug}`}
+        className="py-2 px-3 mx-1 fw-bold text-primary"
+      >
+        CHALLENGES
+      </NavLink>
+      <div className="py-2 px-3 mx-1 fw-bold bg-primary text-white rounded">
+        EXERCISES
+      </div>
```

Comment:
I'm not sure if we want to make this also a NavLink that points to the current page. I made it a div instead since it doesn't seem like there's a point to having a link to the current page, but maybe it makes sense to add a link to the current page. Let me know if you think I should add a link to the current page.

Reply:
Yeah, I agree. It's unnecessary to point to the same page.